src/app/webcontainer/WebApp.qml

```diff
@@ -212,6 +275,7 @@ Common.BrowserView {
                 left: parent.left
                 right: parent.right
                 top: chromeLoader.bottom
+                bottom: parent.bottom
```

Comment:
this has been resolved by https://github.com/ubports/morph-browser/pull/351, binding it to parent.bottom would hide the part of the webview below the on screen keyboard if it is open.